src/main/index.ts

```diff
@@ -123,9 +134,9 @@ app.on("window-all-closed", () => {
       macMenuItem.visible = true;
     }
     return;
+  } else {
+    app.quit();
```

Comment:
Ah since the previous clause actually returns, we don't need the else statement. It's fine to just keep it as `app.quit()` without the else clause.